packages/react/src/internal/components/InputValidation.tsx

```diff
@@ -35,72 +31,33 @@ const InputValidation: React.FC<React.PropsWithChildren<Props>> = ({children, id
   const iconBoxMinHeight = iconSize * captionLineHeight
 
   return (
-    <StyledInputValidation
-      className={clsx({
-        [classes.InputValidation]: enabled,
-      })}
+    <Text
+      className={clsx({[classes.InputValidation]: sx !== defaultSxProp})}
```

Comment:
confused by this line. these styles are only needed if sx prop is supplied?

Reply:
ah I think I was confused here, thinking that we would only apply the modules styles if we didn't have custom `sx`, but I think the idea is that we always apply the styles anyway. I'll make this change!